prow/config.yaml

```diff
@@ -20935,6 +20935,30 @@ periodics:
       secret:
         secretName: service-account
 
+- interval: 24h
+  agent: kubernetes
+  name: periodic-kubernetes-e2e-rpms-pushed
+  spec:
+    containers:
+    - args:
+      - --timeout=60
+      - --repo=k8s.io/kubeadm=master
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
```

Comment:
does this image have the right RPM tools installed?

Reply:
I'm only aware of RPM tools being in `bazelbuild`, we aren't doing anything with RPM in CI currently AFAIK, there are some issues related to this on the backburner.